src/www/ui_components/server-card.html

```diff
@@ -177,6 +185,7 @@
         serverId: String,
         serverHost: String,
         serverPort: Number,
+        usesUnsupportedCipher: Boolean,
```

Comment:
Let's have a generic `errorMsg` property instead. This way we don't attach too much behavior to the value.
Let's also add a `enabled` property that is true by default and we can override to disable it. Or maybe just add a `DISABLED` state

This should simplify the UI.

Reply:
Done. I added both.

For the disabled property, I didn't flush it through all the way to the model. Instead I'm just checking for the existence of an error message :/ Lmk if we should